spec/src/main/asciidoc/required-apis.adoc

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright (c) 2017-2022 Contributors to the Eclipse Foundation
+// Copyright (c) 2017 Contributors to the Eclipse Foundation
```

Comment:
Why remove "2017-2024" and just specify 2017?

Reply:
EF no longer needs to add the change date any more.